### PR TITLE
5.4 Add Ubuntu 18.04 support

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/rs/administering/designing-production/supported-platforms.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Note: This document covers RS 5.0 and up.
 
 | **Platform** | **Versions/Information** |
 |------------|-----------------|
-| Ubuntu |  14.04, 16.04 64-bit, Server or Desktop. The Server version is recommended for production deployments. |
+| Ubuntu (Server or Desktop) |  14.04, 16.04 64-bit, 18.04 64-bit. The Server version is recommended for production deployments. |
```
Comment (Paz-Yanover) [inline content/rs/administering/designing-production/supported-platforms.md]: 1. Move to the last sentence
2. All are 64 bit versions

File: content/rs/getting-started/quick-setup.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ the current directory.
 $ sudo ./install.sh -y
 ```
 
+Note: RS installation on Ubuntu 18.04 fails if port 53 is used by systemd-resolved
```
Comment (Paz-Yanover) [inline content/rs/getting-started/quick-setup.md]: "In default Ubuntu 18.04 installations, port 53 is in use by systemd-resolved (DNS server). In such a case, the system configuration must be changed to make this port available before running RS installation."

File: content/rs/administering/designing-production/supported-platforms.md (hunk 1 above)
Comment (bmansheim) [reply content/rs/administering/designing-production/supported-platforms.md]: Check it out at: https://docs.redislabs.com/staging/5.4-ubuntu-18.04/rs/administering/designing-production/supported-platforms/

File: content/rs/getting-started/quick-setup.md (hunk 2 above)
Comment (bmansheim) [reply content/rs/getting-started/quick-setup.md]: I used:
When port 53 is in use, the installation fails. This is known to happen in default Ubuntu 18.04 installations in which systemd-resolved (DNS server) is running. To workaround this issue, change the system configuration to make this port available before running RS installation.

Comment (bmansheim) [reply content/rs/getting-started/quick-setup.md]: @Paz-Yanover Is this above note OK? Wanted to cover any situation where port 53 is in use.

**Pull request merged**